src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -12128,6 +12191,726 @@ namespace AirflowNetwork {
+                            ShowWarningError(
+                                m_state, "AirflowNetwork Duct Sizing: Duct Sizing Method = PressureLossWithMaximumVelocity for Supply Branch size");
+                            ShowContinueError(
+                                m_state,
+                                format("The Maximum Airflow Velocity at {:.1R} is less than calculated velosity at {:.1R} using PressureLoss",
+                                       AirflowNetworkSimu.DuctSizeMaxV,
+                                       Velocity));
+                            ShowContinueError(m_state, "..The Maximum Airflow Velocity is used to calculate Supply Branch Diameter");
+                        }
+                    }
+                }
+            }
+
+            DuctSizingRTFlag = false;
+            DuctSizingRBFlag = false;
+
+            // return duct trunk
+            if (AirflowNetworkLinkageData(AFNLinkNum).NodeNums[1] == NodeLoopReturn) {
+                SumLength = 0.0;
+                DynamicLoss = 0.0;
+                MaxRough = 0.0;
+                if (AirflowNetworkLinkageData(AFNLinkNum).NodeNums[0] == NodeMixer) {
+                    // A single branch duct
+                    if (CompTypeNum == iComponentTypeNum::DWC && AirflowNetworkLinkageData(AFNLinkNum).ZoneNum > 0) {
+                        AirflowNetworkLinkageData(AFNLinkNum).DuctLineType = iDuctLineType::ReturnTrunk;
+                        TypeNum = AirflowNetworkCompData(CompNum).TypeNum;
+                        SumLength = DisSysCompDuctData(TypeNum).L;
+                        MaxRough = DisSysCompDuctData(TypeNum).roughness;
+                        DynamicLoss = DisSysCompDuctData(TypeNum).TurDynCoef;
+                        DuctSizingRTFlag = true;
+                    }
+                } else {
+                    int NodeNum1 = AirflowNetworkLinkageData(AFNLinkNum).NodeNums[0];
+                    int CompNum1;
+                    iComponentTypeNum CompTypeNum1;
+                    CompNum1 = AirflowNetworkLinkageData(AFNLinkNum).CompNum;
+                    CompTypeNum1 = AirflowNetworkCompData(CompNum1).CompTypeNum;
+                    if (CompTypeNum1 == iComponentTypeNum::DWC) {
+                        AirflowNetworkLinkageData(AFNLinkNum).DuctLineType = iDuctLineType::ReturnTrunk;
+                        TypeNum = AirflowNetworkCompData(CompNum1).TypeNum;
+                        SumLength += DisSysCompDuctData(TypeNum).L;
+                        MaxRough = max(MaxRough, DisSysCompDuctData(TypeNum).roughness);
+                        DynamicLoss += DisSysCompDuctData(TypeNum).TurDynCoef;
+                        DuctSizingRTFlag = true;
+                    }
+                    while (NodeNum1 != NodeSplitter) {
+                        for (AFNLinkNum1 = 1; AFNLinkNum1 <= AirflowNetworkNumOfLinks; AFNLinkNum1++) {
+                            if (NodeNum1 == AirflowNetworkLinkageData(AFNLinkNum1).NodeNums[1]) {
```

Comment:
This is neither here nor there, but rather than deeply nesting conditions within loops, it makes the loop more readable IMO to use `continue`. e.g.:

```cpp
if (NodeNum1 != AirflowNetworkLinkageData(AFNLinkNum1).NodeNums[1])
   continue;

if (AirflowNetworkCompData(CompNum1).CompTypeNum != iComponentTypeNum::DWC)
   continue;

...
```